tests/kernel/mbox/mbox_api/src/test_mbox_api.c

```diff
@@ -480,7 +480,7 @@ void test_mbox_data_get_null(void)
 static void thread_mbox_get_block_data(void *p1, void *p2, void *p3)
 {
 	k_sem_take(&sync_sema, K_FOREVER);
-	struct k_mbox_msg bdmsg;
+	struct k_mbox_msg bdmsg = {0};
```

Comment:
FWIW: I recognize these lines because I just made these elements static in the big xtensa series (they're shared between threads and can't live on the stack when KERNEL_COHERENCE=y).  Looks like I have a rebase collision in my future.

Reply:
Oh, hope this won't  bring you too much trouble. And thanks for let me understand this point, does that mean in the future this would be handled implicitly, or we just make it static for all this? or might be like that:  
```
#if CONFIG_KERNEL_COHERENCE=y
static struct k_mbox_msg bdmsg;
#else
struct k_mbox_msg bdmsg = {0};
#endif
```

Reply:
It was just a passing note.  But yes, if it were static it wouldn't need an initializer to fix this issue.

Reply:
Got it, thanks for notifying this~^^